orca-sql/src/main/kotlin/com/netflix/spinnaker/orca/sql/pipeline/persistence/SqlExecutionRepository.kt

```diff
@@ -1198,34 +1230,8 @@ class SqlExecutionRepository(
       .let { seek(it) }
   }
 
-  private fun DSLContext.selectExecution(type: ExecutionType, fields: List<Field<Any>> = selectFields()): SelectJoinStep<Record> {
```

Comment:
Why did we move these two methods?

Reply:
I moved to `jooq.kt` class because I required these functions in the `ExecutionMapper` class and I want to reuse them.
The ExecutionMapper needs these function because as the PipelineRef feature can work together with the Compression feature the mapper needs to know if it will get the executionParent from the `pipeline` table or the `compressedPipeline` table